dashboard/variable.libsonnet

```
@@ -1,5 +1,9 @@
 {
-  datasource: {
+  datasource_type: {
+    influxdb: 'influxdb',
+    prometheus: 'prometheus',
+  },
+  datasource_var: {
```

```suggestion
  datasource: {
```

After viewing the entire patch, it seems to me it's better to leave the old name. This is not critical, but it seems to me that the new name does not look good in such places:
```
datasource=variable.datasource_var.influxdb,
```
```
addPanels(
  section.cluster_influxdb(
    datasource_type=variable.datasource_type.influxdb,
    datasource=datasource,
    policy=policy,
    measurement=measurement,
  )
)
```

---

Yeah, it's really better. Replaced back again